source/_integrations/schlage.markdown

```diff
@@ -56,3 +68,9 @@ Once you have enabled the Schlage integration, you should see the following swit
 
 - **1-Touch Locking** - When enabled, locks the lock with a press of the Schlage button.
 - **Keypress Beep** - Controls whether the lock will emit beeping tones on use.
+
+## Remove integration
```

Comment:
```suggestion
## Removing the integration
```
I know that's from the example. But I want to change the example to use the gerund, like the other titles for procedural content.